package.json

```diff
@@ -4,7 +4,13 @@
   "description": "Action to handle automatic PR labeling",
   "main": "dist/index.js",
   "scripts": {
-    "build": "ncc build src/index.ts --out dist"
+    "test": "jest",
+    "build": "ncc build src/index.ts --out dist",
+    "prepare": "husky install"
+  },
+  "engines": {
+    "node": "12.x",
+    "npm": "6.x"
```

Comment:
Again here. This should not be specific to npm 6.x, otherwise it's likely going to cause nasty issues down the line.
```suggestion
    "npm": ">=6"
```

Reply:
At that point, since the code itself does not rely on npm in any way, it's better to remove this altogether, and just make a CI task to test that the lockfile is v1 to prevent it being updated within the repo...

Reply:
The npm v6 doesn't fail when the package-lock is v2.

Reply:
The npm v6 won't fail if you use npm v7 to install things, it'll just issue a warning:

```
$ npm --version
np7.12.1

$ npm install
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@tldr-pages/labeler-action@0.1.0',
npm WARN EBADENGINE   required: { node: '12.x', npm: '6.x' },
npm WARN EBADENGINE   current: { node: 'v12.20.0', npm: '7.12.1' }
npm WARN EBADENGINE }

> @tldr-pages/labeler-action@0.1.0 prepare
> husky install

husky - Git hooks installed

added 374 packages, and audited 375 packages in 13s

found 0 vulnerabilities
```

This does not actually stop anyone from converting the package-lock.json to a v2 lockfile and committing it.

Reply:
but you are seeing that warning because I added `npm@6.x`

Reply:
Does not stop me from checking in the v2 lockfile though. Somehow every other JS repo in tldr-pages has gotten by without declaring this field, and yet we _have_ to have it here for some reason.

From the doc on [engines](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines):

> You can also use the "engines" field to specify which versions of npm are capable of properly installing your program.

npm@v7 is perfectly capable of installing the dependencies here, though given the relative simplicity, it's probably true that npm@v4+ could install it right now.